packages/e2e-tests/specs/editor/blocks/list.test.js

```diff
@@ -113,6 +113,9 @@ describe( 'List', () => {
 		await clickBlockAppender();
 		await page.keyboard.press( 'Enter' );
 		await page.keyboard.type( '* ' );
+		// Not entirely certain why  this timeout is needed.
+		// eslint-disable-next-line no-restricted-syntax
+		await page.waitFor( 1000 );
```

Comment:
@ellatrix any ideas here? I suspect the test to be unstable in trunk too, just less visible but I don't understand how this PR impacts this since it's pretty consistent here.

Reply:
I'll look into it